pages/common/wormhole.md

```diff
@@ -0,0 +1,16 @@
+# wormhole
+
+> Get things from one computer to another, safely.
+> More information: <https://magic-wormhole.readthedocs.io/en/latest/>.
+
+- Send a file:
+
+`wormhole send {{file}}`
```

Comment:
How does it know where to send it? Also, can we tweak this to make it more consistent with other pages please?
```suggestion
`wormhole send {{path/to/file}}`
```